root/static/scripts/timeline.js

```diff
@@ -212,13 +212,26 @@ class TimelineViewModel {
 
         _.forEach(parts, function (part) {
             var match;
-            if (match = part.match(/^(-)?([rv])-?$/)) { // trailing - for backwards-compatibility
+
+            /** Please correct me if i'm misunderstanding this
+             * if(match = part.match(RegExp) {
+             *    // Do sth!!
+             * }
+             * 
+             * is equivalent to
+             * 
+             * if(part.match(RegExp)) {
+             *    match = part.match(RegExp);
+             *    // Do sth!!
```

Comment:
Not quite. It's equivalent to:
```JS
match = part.match(RegExp);
if (match) {
  // Do something
}
```
The difference being `part.match` is only executed once, not twice (which doesn't matter in this example, but is very important if the function has side effects).